spec.md

```diff
@@ -87,6 +92,7 @@ can be found under the "omero" key in the group-level metadata:
 ```
 "id": 1,                              # ID in OMERO
 "name": "example.tif",                # Name as shown in the UI
+"version": "0.1",                     # Current version
```

Comment:
:+1: for adding a version attribute. Is there a case for adding it to each spec? I assume this opens the door for incrementing the version of a given spec independently of the rest.
Probably in v0.2 we might want to make this key mandatory

Reply:
A general discussion on how we want to handle the versions (1 vs many; required vs optional) is definitely coming.